### ISSUE TITLE
Internal Video viewer: Videos don't stream

### ISSUE THREAD

**Issue by DudeDudeDudeDudeDude** [nextcloud/android#3622]
### Actual behaviour
- Tell us what happens
Videos don't stream

### Expected behaviour
- Tell us what should happen
Videos should stream 

### Steps to reproduce
I reproduced this on differnet devices:

    1. Moto G5, Android 8.1.0

    2. Sony Xperia Z, Android 6.01

    3. Moto X Play, Android 7.1.1

### Environment data
Android version:
see above

Device model: 
see above
Stock or customized system:

Nextcloud app version:
3.4.2

Nextcloud server version:

#### Web server error log
```
Insert your webserver log here
```

#### Server Configuration 
```

{
    "memcache.local": "\\OC\\Memcache\\APCu",
    "apps_paths": [
        {
            "path": "\/var\/www\/html\/apps",
            "url": "\/apps",
            "writable": false
        },
        {
            "path": "\/var\/www\/html\/custom_apps",
            "url": "\/custom_apps",
            "writable": true
        }
    ],
    "passwordsalt": "***REMOVED SENSITIVE VALUE***",
    "secret": "***REMOVED SENSITIVE VALUE***",
    "trusted_domains": [
        "nx1733.your-next.cloud"
    ],
    "datadirectory": "***REMOVED SENSITIVE VALUE***",
    "dbtype": "mysql",
    "version": "15.0.7.0",
    "overwrite.cli.url": "https:\/\/nx1733.your-next.cloud",
    "dbname": "***REMOVED SENSITIVE VALUE***",
    "dbhost": "***REMOVED SENSITIVE VALUE***",
    "dbport": "",
    "dbtableprefix": "oc_",
    "mysql.utf8mb4": true,
    "dbuser": "***REMOVED SENSITIVE VALUE***",
    "dbpassword": "***REMOVED SENSITIVE VALUE***",
    "installed": true,
    "instanceid": "***REMOVED SENSITIVE VALUE***",
    "redis": {
        "host": "***REMOVED SENSITIVE VALUE***",
        "port": "6379",
        "timeout": "1.5",
        "dbindex": "0"
    },
    "filelocking.enabled": "true",
    "memcache.locking": "\\OC\\Memcache\\Redis",
    "mail_smtpmode": "smtp",
    "mail_smtpauthtype": "LOGIN",
    "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
    "mail_from_address": "***REMOVED SENSITIVE VALUE***",
    "mail_domain": "***REMOVED SENSITIVE VALUE***",
    "log_type": "syslog",
    "syslog_tag": "nextcloud-nx1733",
    "simpleSignUpLink.shown": "false",
    "upgrade.disable-web": "true",
    "updatechecker": "true",
    "twofactor_enforced": "true",
    "twofactor_enforced_groups": [
        "admin"
    ],
    "twofactor_enforced_excluded_groups": [
        "***REMOVED SENSITIVE VALUE***",
        "***REMOVED SENSITIVE VALUE***"
    ],
    "loglevel": 2,
    "maintenance": false
}
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by tobiasKaminsky** [comment]
Depending on the video format it can happen that the internal player is not capable of playing them.
Can you try to stream it with an external app, e.g. VLC?

**Comment by DudeDudeDudeDudeDude** [comment]
Sorry for the late answer. The video formats used are .mov or .mp4. Of course I could try to stream it with VLC. But I think the app should be able to stream the videos itself.

**Comment by tobiasKaminsky** [comment]
If you have a video that you can share with me:
Can you create us a test account, test if the problem occurs also there and if so send the credentials to tobias at nextcloud dot com with a reference to this issue?

**Comment by DudeDudeDudeDudeDude** [comment]
I created a test account for you, put a folder with pictures and a folder with videos in there. The problems occur there as well. Please check it out. I sent you the credentials to your email adress mentioned above.

**Comment by tobiasKaminsky** [comment]
I could successfully play both videos.
However it took quite a long time ~20s to see a video. Before that I only saw "loading".
I tested this on Android 9 emulator.

**Comment by DudeDudeDudeDudeDude** [comment]
I tried this as well. And I can not play neither of the videos. After about 20 seceonds or so, the videos seems to start, but it stops again immediately. It doesn't seem to load at all. The loading bar stops. I attached a screenshot. I'm using Android 8.1.0 on a MotoG5 (XT1676)  
![Screenshot_20190418-114149](https://user-images.githubusercontent.com/47612164/56352705-0d8dcf00-61d0-11e9-8e1e-92bc071d91e9.png)
